jupyterhub/apihandlers/base.py

```diff
@@ -92,10 +92,12 @@ def write_error(self, status_code, **kwargs):
 
     def server_model(self, spawner):
         """Get the JSON model for a Spawner"""
+        is_admin = self.get_current_user().admin
```

Comment:
Let's move this to a False-by-default boolean flag, and pass this as-appropriate on Handlers. See #1817 for an example of such a flag.